CONTRIBUTING.md

```diff
@@ -162,3 +162,18 @@ During the PR process, expect that there will be some back-and-forth. Please try
 If we accept the PR, a [maintainer](MAINTAINERS.md) will merge your change and usually take care of backporting it to appropriate branches ourselves.
 
 If we reject the PR, we will close the pull request with a comment explaining why. This decision isn't always final: if you feel we have misunderstood your intended change or otherwise think that we should reconsider then please continue the conversation with a comment on the PR and we'll do our best to address any further points you raise.
+
+We have a lot of mechanisms to help expedite towards an accepted PR.  Here are some tips for success:
+1. *Minimize BWC guarantees*: First PR review heavily focuses on the public facing API. This is what we have to "guarantee" as non-breaking for bwc across major versions.
+2. *Do not copy ES code*: SSPL co-mingling will get rejected real fast.
```

Comment:
This is incorrect. 

You're not allowed to look at/copy non-APLv2 compatible code, which includes SSPL. You are actually allowed to copy code from ES < 7.10.2 which was APLv2 for example.

So let's stay out of calling out ES explicitly and replace this with something like ensure that code is APLv2 compatible and confirm that you have not copied any code that's not compatible if it implements a feature that may or may not exist in another product?